Makefile

```diff
@@ -0,0 +1,27 @@
+.PHONY: all docs test itest
+
+docs: .taskproc/bin/activate
+	.taskproc/bin/tox -e docs
+
+test: .taskproc/bin/activate
+	.taskproc/bin/tox
+
+itest:
+	cd tests/integration && \
+	docker-compose -f cluster/docker-compose.yaml down && \
```

Comment:
`/bin/sh: docker-compose: command not found`
How about doing all of this within tox? That way all knowledge on how to run tests lives there. As you notice you'll probably want to start a requirements-dev.txt as well.